views/layouts/main.handlebars

```diff
@@ -43,7 +43,7 @@
                         <a class="location" href="https://goo.gl/maps/JIEVs" target="_blank">Dunn Richmond Center - Carbondale, IL</a>
                     </div>
                     <div class="pure-u-1-2">
-                        <a href="/register" class="pure-button pure-button-disabled">Register Soon!</a>
+                        <a href="/register" class="pure-button">Register now!</a>
                         <a href="https://docs.google.com/forms/d/1N4XyY-pbktXrqsyKu0cJC-VHYiQkL62reeEgD3yOD_A/viewform" class="pure-button">Volunteer</a>
```

Comment:
For this can we create a route like `/volunteer` and have that do the redirect? Like we do for `/register`.